x-pack/test/security_solution_endpoint/page_objects/hosts_page.ts

```diff
@@ -121,6 +121,8 @@ export function SecurityHostsPageProvider({ getService, getPageObjects }: FtrPro
       for (let i = 0; i < NodeSubmenuItems.length; i++) {
         await (await testSubjects.findAll('resolver:map:node-submenu-item'))[i].click();
         const Events = await testSubjects.findAll('resolver:map:node-submenu-item');
+        // this sleep is for the AMP enabled run
+        await pageObjects.common.sleep(300);
```

Comment:
How much does this sleep add to the duration of every test?  Is there a way you can convert this to a wait?

Reply:
I try to do the wait and different ways nothing works.

Reply:
it just adds 2 sec per test only

Reply:
I support a Sleep if we need the tests passing and to unblock the build.  Always good to question it and follow up more if we can improve it